global.d.ts

```diff
@@ -19,7 +19,7 @@ export interface PlaywrightMatchers<R> {
   /**
   * Will check if the element's value includes the given text.
   */
-  toHaveText(value: string, options?: PageWaitForSelectorOptions): Promise<R>;
```

Comment:
shouldn't we keep that?

Reply:
@mxschmitt I believe no. We don't have case for that.
https://github.com/playwright-community/expect-playwright/pull/40#issue-555202702